mod_reforged/hooks/skills/actives/split_shield.nut

```diff
@@ -108,6 +115,16 @@
 		return true;
 	}
 
+// New Functions
+	q.RF_getShieldDamage <- function()
+	{
+		if (::MSU.isNull(this.getContainer()) || ::MSU.isNull(this.getContainer().getActor()))
+			return 1; // return a non-zero number so the tooltip gets added
```

Comment:
I am not happy about my solution here to return 1. It will add the damage tooltip to the skill file always - but here we allow with null weapon returning 0 which will remove that entry.

So perhaps it is worth it to assume that this skill will always be attached to a weapon and therefore we can simply always access the weapon's damage and return that?

Reply:
For the beast scenario we want something like `this.m.ShieldDamageOverride` which makes the `RF_getShieldDamage` function ignore anything else and return that value.

Apart from that I think we can expect a weapon to be present.